stream_mem_stress_test/memory.ts

```diff
@@ -0,0 +1,101 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import {
+    KmsKeyringNode,
+    buildClient,
+    CommitmentPolicy
+} from '@aws-crypto/client-node'
+import {AlgorithmSuiteIdentifier} from '@aws-crypto/material-management'
+
+/* This builds client with the REQUIRE_ENCRYPT_REQUIRE_DECRYPT commitment policy.
+ * REQUIRE_ENCRYPT_REQUIRE_DECRYPT enforces that this client only encrypts using committing algorithm suites
+ * and enforces that this client will only decrypt encrypted messages
+ * that were created with a committing algorithm suite.
+ * This is the default if you build `buildClient()`.
+ */
+const { encryptStream, decryptUnsignedMessageStream } = buildClient(
+    CommitmentPolicy.REQUIRE_ENCRYPT_REQUIRE_DECRYPT
+)
+
+import { finished } from 'stream'
+import { createReadStream, createWriteStream } from 'fs'
+import { promisify } from 'util'
+const finishedAsync = promisify(finished)
+
+/* A KMS CMK is required to generate the data key.
+ * You need kms:GenerateDataKey permission on the CMK in generatorKeyId.
+ * This key is public, DO NOT USE in production environment.
+ */
+const generatorKeyId = 
+    'arn:aws:kms:us-west-2:658956600833:alias/EncryptDecrypt'
+
+// configure keyring with CMK(s) you wish to work with
+const keyring = new KmsKeyringNode({ generatorKeyId })
+
+/**
+ * Encryption Context is very useful if you want to assert things about the encrypted data
+ * See: https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/concepts.html#encryption-context
+ */
+const context = {
+    stage: 'demo',
+    purpose: 'streaming memory stress test',
+    origin: 'us-west-2',
+}
+
+/**
+ * kmsEncryptStream will use the encryptStream function and create a pipeline to encrypt a stream of data
+ *  from a file and write it to destination `./{filename}.encrypted`
+ * @param filename string of file name you wish to encrypt
+ * @param framesize optional parameter to determine frame size; default is 4096 bytes
+ */
+export async function kmsEncryptStream(filename:string, framesize?:number) {
```

Comment:
Nit: There should be a space b/w the variable and its type:
```suggestion
export async function kmsEncryptStream(filename: string, framesize?: number) {
```